SDK/MobileCenterCrashes/Microsoft.Azure.Mobile.Crashes.Shared/AndroidErrorDetails.cs

```diff
@@ -4,10 +4,10 @@ namespace Microsoft.Azure.Mobile.Crashes
 {
     public class AndroidErrorDetails
     {
-        public AndroidErrorDetails(object _Exception, string _ThreadName)
+        public AndroidErrorDetails(object throwable, string threadName)
         {
-            Exception = _Exception;
-            ThreadName = _ThreadName;
+            Exception = throwable;
```

Comment:
good first step to rename the parameter but the field must also be renamed to Throwable ;)